R/pip.R

```diff
@@ -172,5 +172,8 @@ pip <- function(country = "all",
   # Select columns
   out <- out[, .SD, .SDcols = lkup$pip_cols]
 
+  #Order rows by country code and reporting year
+  out <- out[order(country_code, reporting_year)]
+
```

Comment:
Since this is a data.table object, I switched to data.table ordering. It is slightly faster, but more importantly, much more consistent:
![image](https://user-images.githubusercontent.com/1552634/168558452-3fa7ea77-aa41-40fc-bfc8-057c93e4c2c5.png)

Reply:
I have also removed the ordering from line ~150 as country_code is not part of the response in this case. This is the response for regional aggregates (Latin America, Sub-Saharan Africa, etc.)